pallets/simple-nft/src/lib.rs

```diff
@@ -56,7 +72,48 @@ pub mod pallet {
     pub struct Pallet<T>(PhantomData<T>);
 
     #[pallet::hooks]
-    impl<T: Config> Hooks<BlockNumberFor<T>> for Pallet<T> {}
+    impl<T: Config> Hooks<BlockNumberFor<T>> for Pallet<T> {
+        fn on_runtime_upgrade() -> frame_support::weights::Weight {
```

Comment:
This looks good but is untested. Have you seen any examples of migration tests you could look at for reference?